winrm/http.go

```diff
@@ -42,7 +42,13 @@ func Http_post(client *Client, request *soap.SoapMessage) (response string, err
 		return
 	}
 	req.Header.Set("Content-Type", soapXML+";charset=UTF-8")
-	req.SetBasicAuth(client.username, client.password)
+
+	if client.useHTTPS == false {
```

Comment:
Same here, you could use a flag for deciding the auth type, instead of using https.

Since they had https before I'm guessing it works with user and password so you shouldn't break that functionality